src/layer/tether.js

```diff
@@ -66,6 +67,9 @@ class Tether extends React.Component<TetherPropsT, TetherStateT> {
   initializePopper() {
     const {placement, popperOptions} = this.props;
     const {modifiers, ...restOptions} = popperOptions;
+
+    if (!this.props.anchorRef || !this.props.popperRef) return;
```

Comment:
why do we need this line?

Reply:
those two props are optional in the component interface, but required to the popper interface